app/views/layouts/application.html.haml

```diff
@@ -31,7 +31,7 @@
     -# Needed for the wicg-inert polyfill. It needs to be on it's own <style> tag, with this `id`
     = vite_stylesheet_tag 'styles/inert.scss', media: 'all', id: 'inert-style'
 
-    -# = preload_pack_asset "locale/#{I18n.locale}-json.js"
+    = vite_preload_file_tag "mastodon/localfes/#{I18n.locale}.json"
```

Comment:
do we want to set the type differently? MDN suggests loading as a fetch type: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/rel/preload#what_types_of_content_can_be_preloaded

Reply:
The end result is
```html
<link rel="modulepreload" href="/packs-dev/intl/en-GB-Dn2xotfG.js" as="script" crossorigin="anonymous" integrity="sha384-nvIcCVX25L5piVIvgc0taZ9ZJCrVekRj561DTTdyRew6yagEtmBNUdt0DwR8MbNE">
```

Even if we import it as a JSON file, Vite transforms it into a plain JS file, so `as="script"` works here. If we want to import it as a JSON file (and call `JSON.parse()` on it), then we need to use a different way to load it